hw2/02-search.js

```diff
@@ -0,0 +1,63 @@
+// Add your code here
+
+const userInput = document.getElementById("userInput");
+const searchBtn = document.getElementById("searchBtn");
+const results = document.getElementById("results");
+const noResults = document.getElementById("noResults");
+
+searchBtn.addEventListener("click", handleClick);
+
+function handleClick() {
+  const query = userInput.value;
+  console.log(query);
+  const search = searchCharacter(query);
+  display(search, query);
+}
+
+function display(characters, query) {
+  results.innerHTML = "";
+  if (characters.length === 0) {
+    noResults.style.display = "block";
+  } else {
+    noResults.style.display = "none";
+    characters.forEach((character) => {
+      const card = createCard(character, query);
+      results.appendChild(card);
+    });
+  }
+}
+
+function createCard(character, query) {
+  const cardDiv = document.createElement("div");
+  cardDiv.classList.add("col-5", "d-inline-flex", "align-items-center");
+
+  const card = document.createElement("div");
+  card.classList.add("card", "h-100");
```

Comment:
Your cards are spaced very oddly. They seem to align to the left and have massive space between the sides but none whatsoever between the tops and bottoms.

Reply:
Fixed!